code/backend/batch/utilities/helpers/azure_search_helper.py

```diff
@@ -121,6 +121,18 @@ def create_index(self):
             ),
         ]
 
+        if self.env_helper.USE_ADVANCED_IMAGE_PROCESSING:
+            logger.info("Adding image_vector field to index")
+            fields.append(
+                SearchField(
+                    name="image_vector",
+                    type=SearchFieldDataType.Collection(SearchFieldDataType.Single),
+                    searchable=True,
+                    vector_search_dimensions=1024,
```

Comment:
thinking aloud: is this something which we want as a constant or a config ?
If constant, can we make it a class level or method level constant ?  Or if config does it also move to ENV ?

Reply:
So I think in future we want to "look up" this value similar to what we do for the `content_vector` dimensions https://github.com/Azure-Samples/chat-with-your-data-solution-accelerator/blob/main/code/backend/batch/utilities/helpers/azure_search_helper.py#L71-L76.

For this though we need to implement the call to the Computer Vision `vectorizeText` end point. That's not something we _need_ until we implement querying.

I suggest once we have that implemented we revist this and dynamically set this field.

Reply:
that's fair. thanks for letting me know.